packages/dx-react-scheduler/docs/reference/grouping-panel.md

```diff
@@ -76,6 +77,8 @@ Field | Type | Description
 group | [Group](#group) | The group the cell represents.
 colSpan | number | The number of columns the cell spans.
 left | number | The cell's offset from the left.
+brightBorderRight? | boolean | Indicates whether to draw a bright right border or an ordinary right border.
```

Comment:
Though `brightRightBorder` sounds slightly unnatural, I think this name better suits here as it consistent with other props such as `hasTightBorder`